### PR TITLE
Helpfile created 

### PR DESCRIPTION
**Pull request by mcjvanschaik**
I have created a helpfile, because after installation, the first thing I did was type :he kiwi. 

Tried to describe the plugin independently and not to repeat the README file. Also I have specified a difference in using <Enter> in visual mode and in normal mode in the README file as well as in the helpfile.

### CODE REVIEW

File: doc/kiwi.nvim.txt
Comment (serenevoid) [inline doc/kiwi.nvim.txt:1]: Can you rename the file to kiwi.txt? That's how I have seen it in most other repos. I'm not very sure about the naming conventions. With this change, I can merge this PR

**Review by serenevoid — CHANGES_REQUESTED** [review]
Rename kiwi.nvim.txt to kiwi.txt

**Comment by serenevoid** [comment]
#14 #15

**Comment by mcjvanschaik** [comment]
I have renamed the file and pushed it to my fork. What do I do now? Do I make a new pull request?

**Review by serenevoid — APPROVED** [review]
Nope. The changes will reflect in this PR when you add the changes to your master branch. Thanks for your contribution!

**Pull request merged**